test_runner.py

```diff
@@ -1,6 +1,6 @@
 import unittest
-from tests import (test_index, test_login, test_session_persistence,
-                   test_rounds, test_rounds_races, test_logout, test_add_round)
+from tests import (test_login, test_session_persistence,
```

Comment:
May as well split the imports on to new lines in this PR to prevent future conflicts.